java/client/src/main/java/glide/api/models/commands/LcsIdxOptions.java

```diff
@@ -0,0 +1,82 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands;
+
+import static glide.utils.ArrayTransformUtils.concatenateArrays;
+
+import lombok.Getter;
+import org.apache.commons.lang3.ArrayUtils;
+
+/**
+ * Optional arguments for TOBEIMPLEMENTED command.
```

Comment:
You can delete this file now, and restore later, when you'll work on TOBEIMPLEMENTED command